public/ims.html

```diff
@@ -83,13 +86,13 @@
       </nav>
    </header>
 
-   <div class="container">
+   <div class="container" id="main-content">
       <h1 class="page-title hyphenate" id="lead">BLE LTER&apos;s Information Management System</h1>
       <br>
       <h2>Our philosophy</h2>
       <p class="lead">The BLE IMS uses modular components and relies on external APIs and open source resources to
          minimize development burden and promote reusability. We welcome discussion and feedback on our system, and we
-         love collaborating with other information managers on improving the state of IM in LTER. We also make sure to follow LTER guidelines and best practices for <a href="https://lternet.edu/wp-content/uploads/2018/05/Guidelines-for-LTER-Web-Site-Design-and-Content-revised-2018-05-17.pdf" target="_blank" rel="noopener">our website</a>, <a href="https://environmentaldatainitiative.org/2018/02/16/eml-best-practices-v-3/" target="_blank" rel="noopener">our metadata</a>, and document all our practices in a <a href="https://github.com/BLE-LTER/ble-handbook" target="_blank" rel="noopener">BLE Information Management Handbook</a>. See <a href="https://github.com/BLE-LTER/" target="_blank" rel="noopener">what we&apos;re up to on GitHub</a>, and
+         love collaborating with other information managers on improving the state of IM in LTER. We also make sure to follow LTER guidelines and best practices for <a href="https://lternet.edu/wp-content/uploads/2018/05/Guidelines-for-LTER-Web-Site-Design-and-Content-revised-2018-05-17.pdf" target="_blank" rel="noopener" aria-label="open in new tab">our website</a>, <a href="https://environmentaldatainitiative.org/2018/02/16/eml-best-practices-v-3/" target="_blank" rel="noopener" aria-label="open in new tab">our metadata</a>, and document all our practices in a <a href="https://github.com/BLE-LTER/ble-handbook" target="_blank" rel="noopener">BLE Information Management Handbook</a>. See <a href="https://github.com/BLE-LTER/" target="_blank" rel="noopener" aria-label="open in new tab">what we&apos;re up to on GitHub</a>, and
```

Comment:
shouldn't the labels say what you're opening in a new tab?  maybe we should view these pages with a screen reader to see what it's like